numpy/lib/npyio.py

```diff
@@ -1549,7 +1549,8 @@ def fromregex(file, regexp, dtype, encoding=None):
 def genfromtxt(fname, dtype=float, comments='#', delimiter=None,
                skip_header=0, skip_footer=0, converters=None,
                missing_values=None, filling_values=None, usecols=None,
-               names=None, excludelist=None, deletechars=None,
+               names=None, excludelist=None,
+               deletechars=NameValidator.defaultdeletechars,
```

Comment:
This is not actually true, as `deletechars` defaults to `None`. I think it belong in the description of `deletechars` below.

Reply:
Sorry wrong comment